misc/docs/source/tutorial/code_snippets/retrieving_data_from_datacenters.rst

```diff
@@ -2,43 +2,123 @@
+The FDSN Web Services
+---------------------
+
++----------------------+--------------------------------+
+| Available Data Types | Format                         |
++======================+================================+
+| Waveforms            | MiniSEED and optionally others |
++----------------------+--------------------------------+
+| Station Information  | StationXML and Text            |
++----------------------+--------------------------------+
+| Event Information    | QuakeML and Text               |
++----------------------+--------------------------------+
 
 .. note::
-    The ``user`` keyword in the following example is used for identification
-    with the ArcLink server as well as for usage statistics within the data
-    center, so please provide a meaningful user id such as your email address.
 
-.. plot:: tutorial/code_snippets/retrieving_data_from_datacenters_1.py
-   :include-source:
+    Not all data providers offer all three data types. Many offer only one or two.
+
+If you want to requests waveforms, or station/event meta information **you will
+most likely want to use the** :mod:`obspy.fdsn` **module**. It is able to
+request data from any data center implementing the `FDSN web services
+<http://www.fdsn.org/webservices/>`_. Data from a large number of centers like
+IRIS/ORFEUS/INGV/ETH/GFZ/RESIF/... (a curated list of data centers implementing
+these services can be found
+`here <http://www.fdsn.org/webservices/datacenters/>`_) can be requested with
+it. As a further advantage it returns data in the most modern and future proof
+formats.
+
+-------
+ArcLink
+-------
+
++----------------------+--------------------------------+
+| Available Data Types | Format                         |
++======================+================================+
+| Waveforms            | MiniSEED, SEED                 |
++----------------------+--------------------------------+
+| Station Information  | dataless SEED, SEED            |
++----------------------+--------------------------------+
+
+ArcLink is a distributed data request protocol usable to access archived
+waveform data in the MiniSEED or SEED format and associated meta information as
+Dataless SEED files. You can use the :mod:`obspy.arclink` module to requests
+data from the `EIDA <http://www.orfeus-eu.org/eida/>`_ initiative but most (or
+all) of that data can also be requested using the :mod:`obspy.fdsn` module.
 
 -----------------
 IRIS Web Services
 -----------------
 
-TODO - for now see :mod:`obspy.iris`
+**Available Data Types and Formats:** Various
+
+IRIS (in addition to FDSN web services) offers a variety of special-purpose web
+services, for some of which ObsPy has interfaces in the :mod:`obspy.iris`
+module. Some of this functionality is meanwhile also directly included in ObsPy
+(e.g. Flinn-Engdahl region lookup in :mod:`obspy.core.util.geodetics`), so
+ObsPy internal tools (not relying on internet connection) can be considered as
+a replacement in some cases.
```

Comment:
There are too many "some"s in this sentence, and I think it could probably be cut in half by removing some redundant parts.